### DISCUSSION TITLE
11ty port?

### DISCUSSION THREAD

**Discussion by antgel** [mmistakes/minimal-mistakes#4825] [General]
Hi there, great to see @iBug picking up this project.

Like a lot of people I guess, I love this theme and used it for my Jekyll site back in the day. However, I've moved to 11ty for new projects.

Some time ago, I started porting the theme to 11ty but the project stalled due to life and now I see there are lots of new changes that I'd have to rebase in. I guess I have two questions:

1. Is there any likelihood of an "official" 11ty version?
2. Looking at recent releases and commits, I get the impression that @iBug is dealing with a large backlog of features and bug fixes that were pending for a long time. Any idea when releases will settle down a bit i.e. I could have another go at developing an 11ty version without worrying about upstream being a significantly moving target?

**Comment by iBug** [comment]
Thanks for your support. I only volunteered to pick it up because I've been using MM for many years extensively on two sites (and for many years to come), one my personal and the other my student club.

1. No. I've never used 11ty, and even if I did, I'd rather start anew with a new theme as porting a theme is going to be an enormous project, since you're dealing in practice with lots of templates if not software (SSG) features at the minimum.

2. As of 4.26 most of the backlog has been cleared, so you can use this version as a starting point. I'm no professional web/frontend designer so chances are not that a large refactoring or rewrite will occur anytime soon. (The only thing in the MM tech stack that I'm familiar with is Jekyll, and Ruby to a minimal extent.) The theme is now in maintenance mode so you should be good to go if you want to port it in its current state.

My advice? Maybe come back to Jekyll, there's a better community around, unless you're good at JS or 11ty itself.
